### PR TITLE
Fix UI when there is not chart maintainer defined

### PR DESCRIPTION
**Pull request by EfrenRey**
@paulino-corujo

### CODE REVIEW

File: src/ui/app/charts/ek-chart-card/ek-chart-card.controller.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ class ChartController {
 
         this.lastModified = $filter('ekHumanizeDate')(this.chart.committed_date, 'epoch');
         this._chartsNavigationActionCreator = chartsNavigationActionCreator;
-        this.maintainer = this.chart.maintainers[0].split('<')[0];
+        this.maintainer = this.chart.maintainers && this.chart.maintainers[0].split('<')[0];
```
Comment (paulino-corujo) [inline src/ui/app/charts/ek-chart-card/ek-chart-card.controller.js]: This breaks the design, I would output something like "Last modified" so the Chart card will display (e.g.)  "Last modified ·5 months ago"

Comment (manolakis) [inline src/ui/app/charts/ek-chart-card/ek-chart-card.controller.js]: I would use

``` javascript
if (!_.isEmpty(this.maintainers) {
    this.maintainer = _.first(this.chart.maintainers).split('<')[0];
}
```

@paulino-corujo ?

File: src/ui/app/admin/ek-admin-charts/ek-admin-charts.controller.js [hunk 2]
```diff
@@ -39,7 +39,7 @@ class AdminChartsController {
                 {
                     name: 'maintainers',
                     field: 'maintainers',
-                    cellTemplate: `<div>{{ item.maintainers[0] }}</div>`,
+                    cellTemplate: `<div>{{ item.maintainers && item.maintainers[0] }}</div>`,
```
Comment (manolakis) [inline src/ui/app/admin/ek-admin-charts/ek-admin-charts.controller.js]: This check is not necessary inside an expression

Comment (EfrenRey) [inline src/ui/app/admin/ek-admin-charts/ek-admin-charts.controller.js]: Thanks

File: src/ui/app/charts/ek-chart-card/ek-chart-card.controller.js (hunk 1 above)
Comment (EfrenRey) [inline src/ui/app/charts/ek-chart-card/ek-chart-card.controller.js]: Changed

**Comment by paulino-corujo** [comment]
ACK

**Pull request merged**